modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/snapshot/SnapshotService.java

```diff
@@ -39,10 +39,7 @@ public void addSnapshotable(Snapshotable snapshotable) {
 
     public byte[] snapshot() {
         HashMap<String, Object[]> snapshots = new HashMap<String, Object[]>(snapshotableList.size());
-
-        if (log.isDebugEnabled()) {
-            log.debug("Taking snapshot ...");
-        }
+        log.info("Taking snapshot ...");
```

Comment:
Thanks for the comment @lasanthafdo, I made this as info logs because when the snapshot is large this log is useful to see whether snapshot is taken or not and whether the delay is due to some other issue.